includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -68,6 +60,40 @@ private function __construct() {
 		}
 	}
 
+	/**
+	 * Handles a few different cases depending on how far along a user is.
+	 * If they have finished the checklist and clicked "I'm done", they get sent to the orders page.
+	 * If they haven't even gone through the wizard yet, they are sent there.
+	 */
+	private function handle_redirects() {
+		if ( empty( $_GET['page'] ) || 'wc-setup-checklist' !== $_GET['page'] ) {
+			return;
+		}
+
+		// If setup has been completed, redirect to orders page.
+		// TODO Redirect to wc-admin once we launch it on eCommerce plans.
+		if ( $this->is_checklist_done() ) {
+			wp_safe_redirect( admin_url( 'edit.php?post_type=shop_order' ) );
+			exit;
+		}
+
+		// If the setup wizard was never complete, redirect to the setup wizard.
+		$notices = WC_Admin_Notices::get_notices();
+		if ( in_array( 'install', $notices ) ) {
+			wp_safe_redirect( admin_url( 'admin.php?page=wc-setup' ) );
```

Comment:
The request came in during a Zoom yesterday. It was mentioned sometimes people make the purchase but for some reason can drop off before making it to the setup wizard, and Jason requested a way that people clicking through from Calypso could get back to the wizard. Since we link to `wc-setup-checklist` from Calypso, this seems like the best way to handle it.

I think it's OK otherwise to not redirect them since it feels like an edge-case.